examples/deployment/deployment.tf

```diff
@@ -1,10 +1,12 @@
 terraform {
+  # The Elastic Cloud provider is supported from ">=0.12"
+  # Version later than 0.12.29 is required for this terraform block to work.
   required_version = ">= 0.12.29"
 
   required_providers {
     ec = {
       source  = "elastic/ec"
-      version = "0.1.0-beta"
+      version = ">=0.1.0-beta"
```

Comment:
This change is making the tf linter unhappy 😄 I think you should change it back to `"0.1.0-beta"`